test/unit/TimedLogoutModal.test.js

```diff
@@ -23,10 +24,11 @@ describe('TimedLogoutModal', () => {
       window.location = { replace: () => {} };
       global.document.cookie = 'accountPageExp=1615496302599';
       sandbox.stub(window.location, 'replace').callsFake(() => { callRecord.push({ replace: true }); });
-      sandbox.stub(global, 'setTimeout').callsFake((fn, timeout) => {
+      timeoutStub = sandbox.stub(global, 'setTimeout').callsFake((fn, timeout) => {
```

Comment:
Do you actually need the `timeoutStub` variable to restore the stub?

Reply:
Without it, just running global.setTimeout.restore() gives this error:      `TypeError: global.setTimeout.restore is not a function`